x-pack/test_utils/lib/utils.ts

```diff
@@ -4,4 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import Chance from 'chance';
+
+const chance = new Chance();
+
 export const nextTick = (time = 0) => new Promise(resolve => setTimeout(resolve, time));
+
+export const getRandomNumber = (range: { min: number; max: number } = { min: 1, max: 20 }) =>
```

Comment:
nit: should the `getRandomString` util be added to this file as well, rather than in `strings.ts`?